src/app/view/Settings.tsx

```diff
@@ -74,47 +79,82 @@ class Settings extends React.Component<Props, State> {
     }
 
     addRound() {
-        helper.setSettings({...helper.getSettings(), rounds: [...this.state.rounds, 60]})
+        helper.setSettings({...helper.getSettings(), rounds: [...this.state.rounds, {duration: 60, breatheInHold: 15}]})
         this.updateRounds();
     }
 
-    setRound(round: number, value: string) {
+    updateRound(round: number, field: string, value: string) {
         const rounds = [...this.state.rounds];
-        rounds[round] = Math.max(0, Number(value));
+        switch(field) {
+            case "duration": {
+                rounds[round].duration = Math.max(0, Number(value));
+                break;
+            }
+            case "breatheInHold": {
+                rounds[round].breatheInHold = Math.max(0, Number(value));
+                break;
+            }
+        }
         helper.setSettings({...helper.getSettings(), rounds: rounds});
         this.updateRounds();
     }
 
-    renderRound(round: number): React.ReactElement {
+    renderRound(roundNumber: number): React.ReactElement {
         return (
-            <div key={round} style={{display: 'flex', justifyContent: 'center', alignItems: 'center'}}>
-                <FontAwesomeIcon
-                    // Hidden icon to ensure centered
-                    style={{visibility: 'hidden', marginRight: '15px'}}
-                    icon={faXmark}
-                />
-                <TextField
-                    style={{backgroundColor: 'white'}}
-                    id='outlined-number'
-                    variant='filled'
-                    label={`Round ${round + 1} retention time (${helper.formatSeconds(this.state.rounds[round])} min)`}
-                    type='number'
-                    value={this.state.rounds[round].toString()}
-                    onChange={(event) => this.setRound(round, event.target.value)}
-                    InputProps={{
-                        endAdornment: <InputAdornment position="start">seconds</InputAdornment>,
-                    }}
-                    InputLabelProps={{
-                        shrink: true,
-                    }}
-                />
-                <FontAwesomeIcon 
-                    style={{visibility: round === 0 ? 'hidden' : 'visible', marginLeft: '15px'}}
-                    className='icon-button' 
-                    icon={faXmark}
-                    onClick={() => this.removeRound(round)} 
-                />
-            </div>
+            <Card variant="outlined" style={{
```

Comment:
I notice the retention time is now duplicated, makes sense to move it to round? 

<img width="863" alt="Screenshot 2023-11-12 at 21 31 09" src="https://github.com/Koenkk/whb/assets/2892853/42349f25-190c-4ce3-acc0-66df016131fa">

Reply:
Good idea - done.